<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Зачислить/Снять</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>190</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>Банкомат С</string>
    </property>
    <widget class="QSpinBox" name="spinBox_C">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99999</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_valueC">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99999</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Сумма&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CC</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Перевод :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>СЧЁТ</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>190</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>Банкомат B</string>
    </property>
    <widget class="QSpinBox" name="spinBox_B">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99999</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_valueB">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99999</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Сумма&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>BA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BC</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Перевод :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>СЧЁТ</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>190</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>Банкомат А</string>
    </property>
    <widget class="QSpinBox" name="spinBox_A">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99999</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_valueA">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99999</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Сумма&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>AA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AC</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Перевод :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>СЧЁТ</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить переводы</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
